web/profiles/custom/yalesites_profile/modules/custom/ys_layouts/templates/layouts/two-column/ys-layouts--yds-twocol.html.twig

```diff
@@ -0,0 +1,24 @@
+{#
+# In this file, we're checking `if directory` in order to determine if the
+# active environment is a Drupal site. We need to do this because Drupal's
+# drag-and-drop interface needs specific attributes on the wrapper elements
+# that would not make sense, or even cause errors in environments like
+# Storybook.
+#}
+
+{% set two_column__base_class = 'yds-two-column' %}
+
+<div {{ attributes.addClass(two_column__base_class) }} data-component-width='max' data-embedded-components>
+    <div {{ attributes.addClass("#{two_column__base_class}__inner") }}>
+        <div {{ region_attributes.primary.addClass("#{two_column__base_class}__primary") }}>
+            {% block two_column__primary %}
+                {{ content.primary }}
+            {% endblock %}
+        </div>
+        <div {{ region_attributes.secondary.addClass("#{two_column__base_class}__secondary") }}>
```

Comment:
I think in this file, we can delete the data-attributes and probably just write the classes manually, rather than using the variable set at the top. (Drupal core doesn't do the variable thing. We do it because we use the BEM function to generate classes, but since this file doesn't necessarily know about our component library etc, this one can be "Drupal standard")